app/components/Wallet/useWallet.tsx

```diff
@@ -10,6 +11,27 @@ import Wallet, {
 	getBalanceMethodName,
 	getNetworkMethodName,
 } from "sats-connect";
+import axios from "axios";
+import { P2WPKH_ADDRESS } from "~/constants";
+
+interface UTXO {
+	scriptpubkey: string;
+	txid: string;
+	value: number;
+	vout: number;
+}
+
+interface ValidateAddressI {
+	isValid: boolean;
+	address: string;
+	scriptPubKey: string;
+	isscript: boolean;
+	iswitness: boolean;
+	witness_version: number;
+	witness_program: string;
+}
+
+const MEMPOOL_API = "https://mempool.space/testnet4/api";
```

Comment:
Please create a task to ensure we choose the right API for the network (testnet, mainnet).

Reply:
https://github.com/gonative-cc/byield/issues/20